### DISCUSSION TITLE
Free drag element or the ability to drag any element anywhere

### DISCUSSION THREAD

**Discussion by CherphAlt** [GrapesJS/grapesjs#4902] [Ideas]
I know that this is possible with canvas i think you can get the x and y position of the object in the canvas and then either use the translate x and y property or calculate the margin top, bottom etc.
This feature will allow users to easily position elements

**Comment by artf** [comment]
There is an absolute dragMode available but there are still a few issues to fix https://github.com/GrapesJS/grapesjs/issues/3770
